roles/kubernetes_monitoring/tasks/main.yml

```diff
@@ -59,6 +59,7 @@
         sample.log: "{{ lookup('file', 'loki-load-test-sample.log') }}"
         ocp-logtest.py: "{{ lookup('file', 'ocp-logtest.py') }}"
   tags: loki-load-test
+  when: loki_load_tests_enabled
```

Comment:
This isn't needed, the state is setup to remove the load test if it is not enabled

Reply:
talked about it and decided to simple put it inside a own role